lib/modules/settingsConsole/templates.js

```diff
@@ -7,7 +7,7 @@ export const consoleContainerTemplate = ({ name, version, gitDescription }: {| n
 	<div id="RESConsoleContainer">
 		<div id="RESConsoleHeader">
 			<div id="RESConsoleTopBar" class="RESDialogTopBar">
-				<div id="RESLogo"></div>
+				<div class="res-logo"></div>
```

Comment:
Doesn't seem strictly necessary but I doubt this is touched by any externally-hosted stylesheet (e.g. subreddit stylesheets, user stylesheets).